server/setup.py

```diff
@@ -47,6 +51,7 @@
     include_package_data=True,
     zip_safe=False,
     entry_points={
+        "console_scripts": ["dive=scripts.cli_dev:cli"],
```

Comment:
This is using the dev cli. We should probably have two entry points for console scripts (one for standard and one for dev) like this:

```python
"console_scripts": ["dive=scripts.cli:cli", "dive-dev=scripts.cli_dev:cli"],
```

Reply:
There's no reason to expose both, IMO.  The dev cli includes everything.  The limited CLI is just so pyinstaller doesn't bundle numpy

Reply:
Got it. In that case, we should probably specify somewhere that the cli requires dev requirements.